kotlin-source/build.gradle

```diff
@@ -32,15 +32,17 @@ dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.11'
 
```

Comment:
And I would also update these IntelliJ run configuration files:

    .idea/runConfigurations/Run_Example_Cordapp___Java.xml
    .idea/runConfigurations/Run_Example_Cordapp___Kotlin.xml
    .idea/runConfigurations/Run_Flow_Tests___Java.xml
    .idea/runConfigurations/Run_Flow_Tests___Kotlin.xml

to help them locate Quasar:
```
<option name="VM_PARAMETERS" value="-ea -javaagent:$PROJECT_DIR$/lib/quasar.jar" />
```